kfs-cam/src/main/java/org/kuali/kfs/module/cab/businessobject/PurchasingAccountsPayableProcessingReport.java

```diff
@@ -23,6 +23,8 @@
 public class PurchasingAccountsPayableProcessingReport extends GeneralLedgerEntry {

```

Comment:
Yes, using edu should work for this case. I didn't extend BO since it could cause class casting exception if instantiating class in org code. But for this Jira, we can proceed with using edu BO since only one place(service code) needs to instantiate BO and it got overridden in edu service. Thus, we got full control of instantiating BO, org or edu class.